test/cypress/support/commands.js

```diff
@@ -113,7 +113,7 @@ Cypress.Commands.add('getByDT', (element, otherSelector = '') => {
 cySbpCheckCommand('giNoPendingGroupKeyShares', (sbp) => {
   const state = sbp('state/vuex/state')
   const pending = Object.keys(state.contracts)
-    .filter(contractID => state[contractID]._vm.type === 'gi.contracts/group')
+    .filter(contractID => state[contractID]._vm?.type === 'gi.contracts/group')
```

Comment:
For some reason, sometimes `_vm` was undefined in the tests, causing this to fail.